ddtrace/contrib/flask/patch.py

```diff
@@ -58,6 +59,69 @@
 flask_version = parse_version(flask_version_str)
 
 
+class _FlaskWSGIMiddleware(_DDWSGIMiddlewareBase):
+    _request_span_name = "flask.request"
+    _application_span_name = "flask.application"
+    _response_span_name = "flask.response"
+
+    # Wrap the `start_response` handler to extract response code
```

Comment:
This comment seems pretty stale here now. It made more sense when the function was being wrapped.

Reply:
```suggestion
```